pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -92,8 +92,7 @@ func (db *DB) init() (err error) {
 		},
 	}
 
-	// TODO pass logger
-	return migration.Run(zap.NewNop(), migrate.NewSqliteDB(db.DB, ""))
+	return migration.Run(zap.L(), migrate.NewSqliteDB(db.DB, ""))
```

Comment:
use `db.log.Named("migrate")`

Reply:
psdb.go doesn't have direct access to `log`

Reply:
Ah, wait... do not change psdb in this PR... that should be a separate PR.

But, if psdb doesn't have access to log yet, then it should be passed down somehow.

Reply:
ok, can be separte PR. Later we can pass logger down.